apps/condo/domains/ticket/schema/TicketReportService.js

```diff
@@ -38,29 +65,34 @@ const TicketReportService = new GQLCustomSchema('TicketReportService', {
             access: true,
             type: 'type TicketReportWidgetOutput { data: [ TicketReportData! ] }',
         },
+        {
+            access: true,
+            type: `enum TicketType { ${TICKET_TYPES.join(' ')} }`,
```

Comment:
We already have this type inside ticket status

Reply:
@pahaz Not certainly in that way. This enum describes flags for Ticket entity like `isEmergency` & `isPaid`

Reply:
Ticket model has isEmergency and isPaid as flags.

Reply:
Please avoid creating any new terminology. Use existing solutions